pysap/plugins/mri/reconstruct_3D/fourier.py

```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
+##########################################################################
+# XXX - Copyright (C) XXX, 3017
+# Distributed under the terms of the CeCILL-B license, as published by
+# the CEA-CNRS-INRIA. Refer to the LICENSE file or to
+# http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+# for details.
+##########################################################################
+
+"""
+Fourier operators for cartesian and non-cartesian space.
+"""
+
+
+# Package import
+import warnings
+from .utils import convert_locations_to_mask_3D
+from pysap.plugins.mri.reconstruct.fourier import FourierBase
+from modopt.interface.errors import warn
```

Comment:
Why using the modopt warn?
Can we use the one from the warnings package?

Reply:
the one from Modopt has a nice color wrapping + doesnt copy the line of the warning. I can change if necessary